mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.tmpl

```diff
@@ -4862,13 +4886,12 @@ resource "google_container_node_pool" "np" {
     oauth_scopes = [
       "https://www.googleapis.com/auth/cloud-platform",
     ]
-    machine_type = "n1-standard-8"
```

Comment:
pulling all these out because a) was seeing a lot of stockout issues with n1, b) n1 is super old, c) n1 isn't needed in any of these for a reason, and d) 

I imagine it probably would be good to (separately) adjust to the n1 stuff in these tests to e2 or n2/n3? There are some comments about e2 not working with `pd-ssd`, but afaict, it works  fine now

Reply:
Yeah, we've got other ticking time bombs like this scattered throughout our tests. The challenge is that APIs require that we specify a machine type/disk type/Linux distro version/etc. and it's often the case that the best we can do is use the latest available to give us the maximum useful lifetime for the test and then fix it when it inevitably breaks in a few years.

Reply:
@SirGitsalot some of the comments imply it's because e2 (which I think is the default?) is required for certain tests (SSD is one example given). I think in some cases, those limitations have changed and / or the code was just copy / pasted.

Taking it out here seemed to work in my test, but happy to adjust it to a different explicit type if needed.